test/unit/messaging/send-message.test.js

```diff
@@ -0,0 +1,58 @@
+const mockSendMessage = jest.fn()
+const mockCloseConnection = jest.fn()
+
+const mockMessageSender = jest.fn().mockImplementation(() => {
+  return {
+    sendMessage: mockSendMessage,
+    closeConnection: mockCloseConnection
+  }
+})
+
+jest.mock('ffc-messaging', () => {
+  return {
+    MessageSender: mockMessageSender
+  }
+})
+
+jest.mock('../../../app/messaging/create-message')
+const { createMessage: mockCreateMessage } = require('../../../app/messaging/create-message')
+
+const { TYPE } = require('../../../app/constants/type')
+const { MESSAGE_ID } = require('../../mocks/messaging/message-id')
+
+const { BODY } = require('../../mocks/messaging/body')
+const { RESPONSE_MESSAGE } = require('../../mocks/messaging/message')
```

Comment:
Mock is with the constant rather than the other mocks, but other than that I'm happy to approve this PR